src/sass/chrome.scss

```diff
@@ -47,145 +41,67 @@ aside {
 
 // Page temp styles
 .pf-c-page {
-    --pf-c-page__sidebar--Transition: all 500ms cubic-bezier(.42, 0, .58, 1);
-    --pf-m-emphasis--FontSize: var(--pf-global--FontSize--xl);
-
-    .ins-c-chrome__global-filter {
-        padding: var(--pf-global--spacer--md) var(--pf-global--spacer--lg);
-        background: var(--pf-global--BackgroundColor--100);
-
-        .pf-c-chip-group { margin-right: var(--pf-global--spacer--xs); }
-        .ins-m-sticky { background: var(--pf-global--palette--blue-50); }
-        .ins-m-permanent button { display: none; }
-        .pf-l-split__item:first-of-type { min-width: 150px;}
-
-        .ins-c-tagfilter {
-            width: 100%;
-
-            &.pf-m-expanded { width: 450px; }
-            .pf-c-select__menu {
-                max-height: 400px;
-                overflow: auto;
-            }
-
-            .pf-c-select__menu li button:hover label .pf-c-badge.pf-m-read {
-                background: var(--pf-global--Color--light-100);
-            }
-
-            .ins-c-tagfilter__option-value {
-                max-width: 360px;
-            }
-        }
-    }
+  --pf-c-page__sidebar--Transition: all 500ms cubic-bezier(.42, 0, .58, 1);
```

Comment:
Mhh, I know the timing is a bit different, but do you think we could use `--pf-global--TimingFunction` for the animation function? At least we'll be aligned with PF.

Reply:
@karelhala Actually, we didn't need that line at all. We were overriding the default timing.